web/db.py

```diff
@@ -55,7 +55,7 @@
 
 # Supported db drivers.
 pg_drivers = ["psycopg2"]
-mysql_drivers = ["pymysql", "mysql.connector"]
+mysql_drivers = ["pymysql", "MySQLdb", "mysql.connector"]
```

Comment:
```suggestion
mysql_drivers = ("pymysql", "MySQLdb", "mysql.connector")
```

Tuples are smaller / more efficient than lists and this signals to the reader that the collection will not change at runtime.

Reply:
Fixed.